### ISSUE TITLE
Styled Icons dont install

### ISSUE THREAD

**Issue by LaksCastro** [jacobwgillespie/styled-icons#1134]
I tried to install Styled Icons using Yarn and this error happens:
```
yarn add styled-icons
yarn add v1.21.1
[1/4] Resolving packages...
[2/4] Fetching packages...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
error An unexpected error occurred: "https://registry.npmjs.org/styled-icons/-/styled-icons-9.2.0.tgz: ESOCKETTIMEDOUT".
info If you think this is a bug, please open a bug report with the information provided in "C:\\Users\\castr\\Desktop\\moria\\yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```
I'm using Gatsby Starter Default and I realized that this error only happens in the latest versions of Styled Icons
I already checked the internet, and everything is fine.

A complete log is [available here](https://gist.github.com/LaksCastro/8a81938be6d4c3059ba51b6b4a653d8b)

**Comment by gocreating** [comment]
Same problem. The installation keeps fail.

**Comment by jacobwgillespie** [comment]
This is an issue with Yarn and Windows that has been seen before (#420, #785) - you may want to report this issue in the Yarn issue tracker.  In the meantime, you may have more luck installing a smaller set of the icons like so:

```shell
$ yarn add @styled-icons/material
```

```typescript
import {Alert} from '@styled-icons/material'
```

That will install only that specific icon pack, and it's more likely to install on Windows.

Alternatively you may find the upcoming Yarn 2.0 to fix this.

**Comment by pak11273** [comment]
```yarn add styled-icons --network-timeout 100000```
